Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -160,12 +158,45 @@ public struct BuildDescription {
 
     /// The inputs of the build plan so we don't need to re-compute them  on every call to
     /// `fileAffectsSwiftOrClangBuildSettings`.
-    private let inputs: [BuildPlan.Input]
+    private let inputs: [Build.BuildPlan.Input]
+
+    public static func load(
```

Comment:
issue (blocking): Since we are introducing a new API, it would be ideal to create some contract tests to ensure we don't regress on the behaviour.

Reply:
See my original comment:
> Still have to update the tests here, just wanted to kick off the SourceKit LSP tests before the weekend.

But I should have made this a draft. It was late and I just wanted to kick things off 😅